release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -1183,19 +1183,48 @@ As part of the continued deprecation of the Operator package manifest format, th
+
+* Previously, HTTP responses were not closed properly by the machine-api-termination-handler. Consequently, goroutines were leaked in `net.http` read and write loops, which led to high memory usage. This update ensures that HTTP responses are always closed properly. As a result, memory usage is now stable. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1934021[*BZ#1934021*])
+
+* Previously, multiple client sets created inside of the MachineSet controller caused slow startup times, which resulted in pods failing readiness checks in some large clusters. Consequently, the MachineSet controller would get stuck in an endless loop. This update fixes the MachineSet controller so that it uses a single client. As a result, the MachineSet controller behaves as expected. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1934216[*BZ#1934216*])
+
+* Previously, instances would take longer to boot when an upgrade was performed by the Machine Config Daemon on the first boot. Consequently, worker nodes would get stuck in restart loops, and machine healthchecks (MCHs) would remove the worker nodes because they did not properly start. With this update, MHCs no longer remove nodes that have not started correctly. Instead, MHCs only remove nodes when explicitly requested. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1939054[*BZ#1939054*])
+
+* Previously, a certificate signing request (CSR) approval was delayed for an unknown reason. Consequently, new machines appearing in the cluster during installation were not approved quick enough, which would prolong cluster installation. This update increases the cache resync period to 10 minutes, instead of 10 hours, to mitigate occasional API server unavailability in early installation phases. As a result, master machines are now approved faster so that cluster installation is no longer prolonged. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1940972[*BZ#1940972*])
+
+* Previously, the default Google Cloud Platform (GCP) image was out of date and referenced a version from the 4.6 release that did not support newer Ignition versions. Consequently, new machines could not boot in 4.7+ clusters that used the default GCP image. With this update, the GCP image is updated to match the release version. As a result, new machines can now boot with the default GCP image. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1954597[*BZ#1954597*])
 
+* Previously, due to a strict check of the virtual machine's (VM) 'ProvisioningState' value, the VM could fail during an existence check. With this update, the check is more lenient so that only deleted machines go into a `Failed` phase during an existence check procedure. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1957349[*BZ#1957349*])
```

Comment:
```suggestion
* Previously, due to a strict check of the virtual machine's (VM) `ProvisioningState` value, the VM would sometimes fail during an existence check. With this update, the check is more lenient so that only deleted machines go into a `Failed` phase during an existence check. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1957349[*BZ#1957349*])
```